gradient-boosting/script.whizzml

```diff
@@ -42,21 +42,21 @@
 
 ;; Helper methods to get dataset attributes
 (define (get-fields dataset)
-  (get (fetch dataset {"limit" -1}) "fields"))
+  ((fetch dataset) "fields"))
```

Comment:
Does (fetch dataset) retrieve all the fields structure? (no field number limits?). That's what limit -1 is here for, I guess...